README.md

```diff
@@ -56,6 +56,10 @@ maven_install(
     fail_on_missing_checksum = False,
     # Fetch srcjars. Defaults to False.
     fetch_sources = True,
+    # Exclude packages from being pulled in by automatic dependency resolution
+    excluded_artifacts = [
```

Comment:
Please combine this with https://github.com/bazelbuild/rules_jvm_external#artifact-exclusion